src/store/bookPage/selectors.js

```diff
@@ -0,0 +1,3 @@
+export const selectBookDetails = (state) => {
+  return state.bookPage;
+};
```

Comment:
small stuff, but if you open an arrow function and immediately return, its better to write it in automatic return style
```
export const selectBookDetails = (state) => state.bookPage
```